<launch>	
	<!-- Makes pedestrian detector node. Change between the good and the bad detector by changing the model dir. -->
	<node type="pedestrian_detector_node" name="ped" pkg="pedestrian_detector" respawn="true" output="screen">
		<param name="model_dir" value="$(find pedestrian_detector)/pedmodels/LdcfInriaDetector.mat" />
		<!--LdcfInriaDetector.mat or  AcfInriaDetector.mat /-->
		<param name="imageResize" value="0.5" />
		<param name="cameraHeight" value="0.27" />
		<param name="angleTiltDegrees" value="0.0" />

		<!-- Change name topic_name to match topic /-->

		<param name="topic_name" value="/usb_cam/image_raw" />
	</node>


	<!-- Makes viewing node for showing detected pedestrians. -->
	<node name="image_view" pkg="image_view" type="image_view" respawn="false" output="screen">
			<remap from="image" to="/ped/imageWithBBox"/>
			<param name="autosize" value="false" />
	</node> 

	<
</launch>
